app/views/publisher_information/index.govspeak.md

```diff
@@ -0,0 +1,9 @@
+####Scheduling
+**22 October 2018**
```

Comment:
These don't seem to be compatible with the styling in the design. We might need to split this out to a YAML file to produce something consistent with the design.

Reply:
It's worth asking if the grey text is worth changing the emphasis of management from YAML to MD files.  By making this page depend more on YAML properties, we make the maintenance of this page different than the other two pages for beta-capabilities and how-to.

Reply:
Sure, I think it'll go that way eventually so it depends how big a job you think it is.

In the meantime it might well work if we insert HTML in there by doing something like `<span class="govuk-hint">22 October 2018</span>`